doc/classes/MultiMeshInstance3D.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="MultiMeshInstance3D" inherits="GeometryInstance3D" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Node that instances a [MultiMesh].
+		Node that instantiates a [MultiMesh].
```

Comment:
Changes in this file should be reverted as "instance" is the correct terminology here